packages/react-ai/.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  extends: ['@aws-amplify/amplify-ui/react'],
+  ignorePatterns: [
+    '.eslintrc.js',
+    'dist',
+    'rollup.config.ts',
+
+    // NOTE: remove to turn on linting for test files
+    '__tests__',
```

Comment:
Let's turn this on now. The only reason this line is in the react package was due to how many issues existed in the test files before linting was added